en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md

```diff
@@ -0,0 +1,205 @@
+          - cp_admin_pwd=admin
+          - adapter_admin_pwd=admin
+        ports:
+          - "18000:18000"
+          - "9843:9843"
+        links:
+          - apim
+      enforcer1:
+        image: wso2/choreo-connect-enforcer:0.9.0
+        logging:
+          options:
+            max-size: "20m"
+            max-file: "5"
+        volumes:
+          - ../resources/enforcer/security:/home/wso2/security
+          - ./conf/log4j2.properties:/home/wso2/conf/log4j2.properties
+          - ../resources/enforcer/dropins:/home/wso2/lib/dropins
+        links:
+          - adapter
+          - apim
+        environment:
+          - ENFORCER_PRIVATE_KEY_PATH=/home/wso2/security/keystore/mg.key
+          - ENFORCER_PUBLIC_CERT_PATH=/home/wso2/security/keystore/mg.pem
+          - TRUSTED_CA_CERTS_PATH=/home/wso2/security/truststore
+          - ADAPTER_HOST_NAME=adapter
+          - ADAPTER_HOST=adapter
+          - ADAPTER_XDS_PORT=18000
+          - ENFORCER_LABEL=choreo-connect-2
+          - ENFORCER_REGION=UNKNOWN
+          - XDS_MAX_MSG_SIZE=4194304
+          - XDS_MAX_RETRIES=3
+          - JAVA_OPTS=${JAVA_OPTS} -Dhttpclient.hostnameVerifier=AllowAll
+          - apim_admin_pwd=admin
+          - enforcer_admin_pwd=admin
+          - tm_admin_pwd=admin
+          - analytics_authURL=https://localhost:8080
+          - analytics_authToken=
+        ports:
+          - "8081:8081"
+      enforcer2:
+        image: wso2/choreo-connect-enforcer:0.9.0
+        logging:
+          options:
+            max-size: "20m"
+            max-file: "5"
+        volumes:
+          - ../resources/enforcer/security:/home/wso2/security
+          - ./conf/log4j2.properties:/home/wso2/conf/log4j2.properties
+          - ../resources/enforcer/dropins:/home/wso2/lib/dropins
+        links:
+          - adapter
+          - apim
+        environment:
+          - ENFORCER_PRIVATE_KEY_PATH=/home/wso2/security/keystore/mg.key
+          - ENFORCER_PUBLIC_CERT_PATH=/home/wso2/security/keystore/mg.pem
+          - TRUSTED_CA_CERTS_PATH=/home/wso2/security/truststore
+          - ADAPTER_HOST_NAME=adapter
+          - ADAPTER_HOST=adapter
+          - ADAPTER_XDS_PORT=18000
+          - ENFORCER_LABEL=choreo-connect-2
+          - ENFORCER_REGION=UNKNOWN
+          - XDS_MAX_MSG_SIZE=4194304
+          - XDS_MAX_RETRIES=3
+          - JAVA_OPTS=${JAVA_OPTS} -Dhttpclient.hostnameVerifier=AllowAll
+          - apim_admin_pwd=admin
+          - enforcer_admin_pwd=admin
+          - tm_admin_pwd=admin
+          - analytics_authURL=https://localhost:8080
+          - analytics_authToken=
+        ports:
+          - "8082:8081"
+    ```
+    
+    !!! note
+        If you are configuring a distributed adapter deployment, you need to define the separate adapters along with the routers and the enforcers deployed in the same environment.
+
+3. Add the gateway environments from the Gateways in the APIM admin portal.
```

Comment:
mention the environment names in this step (choreo-connect-1, choreo-connect-2)

Reply:
Added with https://github.com/wso2/docs-apim/pull/3839/commits/618a329d1dd1ef57172162610208c09d8c6d4c99